io/disk/port_bounce.py

```diff
@@ -146,25 +152,27 @@ def porttoggle(self, test_ports, sleep_time):
         except CommandFailed as cf:
             self.log.info("port disable failed for port(s) %s, details: %s",
                           test_ports, str(cf))
-        time.sleep(sleep_time)
```

Comment:
Why not use this sleep time everywhere ?

Reply:
@narasimhan-v this sleep time is defer for different scenarios.

Reply:
@Naresh-ibm having different sleep times everywhere is not ideal.

1. We should try to eliminate the use of sleep. probably you can explore the wait_for in utils
2. If 1 does not work good for a situation, have sleep time but uniform everywhere.
3. If 2 also does not work good for a situation, specify why a different sleep time is needed. I am against 3, since it would be different for different infrastructure (ex: different OS, different number of multipath devices, etc)

Reply:
@narasimhan-v  this sleep time is given intentionally for port bounce, (for short port bounce and for long port bounce), which will be in minutes and hence we cannot use same variable as other sleep time. hope you got it.

Reply:
Ok, Sorry. I got confused with port bounce time with sleep time.
Can you make all the other sleep times uniform and use it via a variable please